.gitignore

```diff
@@ -66,3 +66,5 @@ db.sqlite3
 /data/
 src/backend/joanie/static/joanie/
 
+# # IDEs
+.idea/*
```

Comment:
Commit c1c01c3531cea42cdd184b368eb98becaa07204e can be dropped since PR #234 has been merged